api/types/session.go

```diff
@@ -84,6 +82,8 @@ func NewWebSession(name string, kind string, subkind string, spec WebSessionSpec
 		},
 		Spec: spec,
 	}
+	session.Metadata.SetExpiry(spec.Expires)
```

Comment:
I believe I misinterpreted `Metadata.Expires` and `Session.Spec.Expires`. Metadata.Expires is meant as a resource expiry in backend context while `Spec.Expires` declares the general expiration and these might differ - I'll update.

Reply:
No, I believe this is a correct assumption. I have not found a counter argument in the current code base and it seems that other resources (i.e. semaphores) default to the same behavior.